src/app/features/inherited-field/admin-inherited-field/admin-inherited-field.component.spec.ts

```
@@ -0,0 +1,46 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { MatDialog } from "@angular/material/dialog";
+import { entityRegistry, EntityRegistry } from "../../../core/entity/database-entity.decorator";
+import { EntityRelationsService } from "../../../core/entity/entity-mapper/entity-relations.service";
+import {
+  entityRegistry,
```

Biome (2.1.2)

[error] Shouldn't redeclare 'entityRegistry'. Consider to delete it or rename it.

'entityRegistry' is defined here:


(lint/suspicious/noRedeclare)